.github/workflows/codeql-analysis.yml

```diff
@@ -24,11 +24,17 @@ jobs:
       matrix:
         # Override automatic language detection by changing the below list
         # Supported options are ['csharp', 'cpp', 'go', 'java', 'javascript', 'python']
-        language: ['csharp', 'javascript']
+        language: ['csharp']
```

Comment:
Are we OK not scanning JavaScript? I notice we have 3 JS files in our repo: <https://github.com/microsoft/Omex/search?l=javascript>

Reply:
I've seen those but I'm not sure about their function I think it's some kind of samples for DocumentDb scripts and they might not play by the regular js rules. I think we could add js back at any moment, for now, I don't see a lot of usefulness in it.

Reply:
Makes sense. I think they're also unlikely to change over time.

It would be great though if we could find out what they are and document them so that others could use them. Not something for this PR though.